### DISCUSSION TITLE
Su doldurmaca

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#53] [Zeka Sorusu] [answered]
Elimizde biri 9 litre diğeri de 4 litre hacimli iki kova ve sınırsız su var. Sadece bu kovaları kullanarak kovaların birinde (tabii ki 9 litre olanı) tam 6 litre su elde edin.

**Comment by OmerAkbayrak** [comment, marked as answer]
**Önce 9 litrelik kovayı doldururuz sonra o kovadan aldığı kadarını 4 litrelik kovaya boşalttığımız zaman 9 litrelik kovada 5 litre kalır sonra 4 litrelik kovadaki suyu atarız boşalan 4 litrelik kovaya 9 litrelik kovadan tekrar su dökeriz ve 9 litrelik kovada 1 litrelik su kalır 4 litrelik kovadaki suyu tekrar atarız ve 9 litrelik kovadaki 1 litre suyu 4 litrelik kovaya boşaltırız. Sonra boşalmış olan 9 litrelik kovayı tekrar doldururuz  ve içinde 1 litre su olan 4 litrelik kovaya aldığı kadarını dökeriz. 3 litreden fazla alamayacağı için 9 litrelik kovada 6 litre su kalmış olur.**
